### DISCUSSION TITLE
Need to Implement the product tour for every enhancement in product

### DISCUSSION THREAD

**Discussion by Mounikach-heeddata** [gilbarbara/react-joyride#1106] [Q&A] [answered]
Need to Implement the product tour for each enhancement that has been implemented newely in product or application after every release it need to update .suggest me how to implement it. And also it should come in which are the pages enhancenment done and also to revisit the tour.

**Comment by gilbarbara** [comment, marked as answer]
Hey @Mounikach-heeddata 

This is a big request. Have you checked the docs and the examples?
Keeping the saved state is a very subjective problem. It depends on your setup and choice of persisting it. It could be locally or in your database, preferably.

**Comment by Mounikach-heeddata** [comment]
Thank you @gilbarbara 

I am able to store my values in local storage , can you suggest me how can i show if any value in tha local storage changes that should be shown as a tour , can you suggest how can i implement that tour ,i.e it  should show that particular thing i have been changed.
(e.g : If i change some column like user to userName then it should show  column name user has been changed to userName in the tour )

**Comment by gilbarbara** [comment]
Hey @Mounikach-heeddata 

You can compare the actual steps with your saved data and filter the steps to include only the unseen steps.
How you will compare the data and which steps to render is exclusive to your app, and I can't tell you how to do it.

Good luck!
